contrib/stack/topsStack/Stack.py

```diff
@@ -922,7 +922,7 @@ def get_orbit(self, orbitDir, workDir):
            orbit_start_date_time = datetime.datetime.strptime(fields[6].replace('V',''), datefmt)
```

Comment:
Maybe its better to add a minute using datetime.timedelta(seconds=60) to orbit_start_date_time and subtracted from orbit_stop_date_time - this will make it more resilient to scenes whos start times and end times coincidentally are same as the orbit file

Reply:
@piyushrpt I see, I'll also add a margin for the precise orbits as advised. Do you mean a change similar to: https://github.com/isce-framework/isce2/blob/fb7ad09211b6cd2fc51b8da9e87eb5f4aab77824/components/isceobj/Sensor/TOPS/Sentinel1.py#L847-L848
orbit_start_date_time - margin
orbit_stop_date_time + margin


Instead of:
orbit_start_date_time + margin
orbit_stop_date_time - margin

Reply:
I think you have the sign wrong. You want to reduce the validity of time period covered by an orbit file, to ensure that there is sufficient padding to get state vectors corresponding to start time / end time of the burst. You want to delay the orbit start time determined by file name, so add margin and take out the last bits from the orbit end time determined from filename.

The code snippet you are pointing to does the opposite, by design, as it looks for state vectors that span the image - so it expands the scene time limits. The two scenarios should use opposite signs on the margins.

Reply:
Again, these will only get used in edge cases that may not even be encountered. The other changes beside the margin look good. So, if you can fix the sign on those or remove it - either is fine.

Reply:
Thanks for clarifying, I've tested the margin changes with pre-downloaded orbits okay, and just pushed the changes.